Core/OxidEE_Events.php

```diff
@@ -160,38 +157,26 @@ private static function _createOrderTable()
     private static function _createOrderTransactionTable()
     {
         $sQuery = "CREATE TABLE IF NOT EXISTS `wdoxidee_ordertransactions`(
-            `wdoxidee_transactionid` int(11) unsigned NOT NULL AUTO_INCREMENT,
-            `wdoxidee_orderid` char(32) character set latin1 collate latin1_general_ci NOT NULL,
-            `wdoxidee_requestid` varchar(256) NOT NULL,
-            `wdoxidee_transactiontype` varchar(128) NOT NULL,
-            `wdoxidee_amount` decimal(9,2) NOT NULL,
-            `wdoxidee_refundedamount` decimal(9,2) NOT NULL,
-            `wdoxidee_currency` varchar(32) NOT NULL,
-            `wdoxidee_transactiondate` datetime NOT NULL,
-            `wdoxidee_transactionstatus` enum('pending','success','error') NOT NULL DEFAULT 'pending',
-            `wdoxidee_timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP,
-            `wdoxidee_comment` mediumtext,
-            PRIMARY KEY (`wdoxidee_transactionid`),
-            KEY `wdoxidee_orderid` (`wdoxidee_orderid`),
-            KEY `wdoxidee_transactiondate` (`wdoxidee_transactiondate`)
+            `OXID` char(32) NOT NULL,
+            `WDOXIDEE_ORDERID` varchar(32) NOT NULL,
+            `WDOXIDEE_TRANSACTIONID` varchar(32) NOT NULL,
+            `WDOXIDEE_PARENTTRANSACTIONID` varchar(32),
+            `WDOXIDEE_REQUESTID` varchar(32) NOT NULL,
+            `WDOXIDEE_ACTION` enum('purchase','authorization') NOT NULL DEFAULT 'purchase',
+            `WDOXIDEE_STATE` enum('success','error','pending'),
```

Comment:
We are using in opencart awaiting, success, and closed. Please adopt.

Reply:
This has been adjusted in the PR that handles the PayPal notification (WDPD-24). Is it okay if we leave it in that PR and keep it as it is?

https://github.com/wirecard/oxid-ee/pull/20/files#diff-bf58dad8826516f4deb6662822da7057R193

Reply:
For me its ok, but you will not be able to do a approval for the US. The BA will not approve the US until the statuses will be correct.

Reply:
@tomazpu added a subtask to our PR feedback Jira ticket that will be addressed in (WDPD-24):
https://jira.parkside.at/browse/WDPD-92